src/taoensso/sente.cljc

```diff
@@ -1339,15 +1341,18 @@
      (let [packer (coerce-packer packer)
 
            win-loc (enc/get-win-loc)
-           win-protocol  (:protocol win-loc)
+           protocol (case (keyword protocol)
+                      :wss "https:"
+                      :ws  "http:"
+                      (:protocol win-loc))
```

Comment:
If we're going to allow the user to specify a protocol, then that should go first. Probably an `or` here would be appropriate?